config.lua.dist

```diff
@@ -96,6 +96,7 @@ premiumToSendPrivate = false
 forceMonsterTypesOnLoad = true
 cleanProtectionZones = false
 showPlayerLogInConsole = true
+checkDuplicatesValues = false
```

Comment:
I think this is not clear enough. How about `checkDuplicateStorageKeys`? Also, some description would be cool, but a name change is enough

Reply:
Done!